### PR TITLE
Add code for GtfToRefflat

### PR DESCRIPTION
**Pull request by ffinfo**

### CODE REVIEW

File: src/main/scala/nl/biopet/tools/gtftorefflat/GtftoRefflat.scala [hunk 1]
```diff
@@ -1,17 +1,186 @@
...
+      gene <- genes.sortBy(_.start)
+      transcript <- gene.transcripts
+    } {
+      val exons = transcript.exons.sortBy(_.start)
+      val values = List(
+        gene.name,
+        transcript.name,
+        gene.contig,
+        if (gene.strand) "+" else "-",
+        (transcript.transcriptionStart - 1).toString, //TODO: check if this is correct
```
Comment (rhpvorderman) [inline src/main/scala/nl/biopet/tools/gtftorefflat/GtftoRefflat.scala]: Is this correct?

File: src/main/scala/nl/biopet/tools/gtftorefflat/GtftoRefflat.scala [hunk 2]
```diff
@@ -1,17 +1,186 @@
...
+        transcript.transcriptionEnd.toString,
+        transcript.codingStart
+          .getOrElse(transcript.transcriptionEnd)
+          .toString, //TODO: check if this is correct
+        transcript.codingEnd
+          .getOrElse(transcript.transcriptionEnd)
+          .toString, //TODO: check if this is correct
+        transcript.exons.length.toString,
+        exons.map(_.start - 1).mkString("", ",", ","), //TODO: check if this is correct
```
Comment (rhpvorderman) [inline src/main/scala/nl/biopet/tools/gtftorefflat/GtftoRefflat.scala]: Is this correct?

**Review by rhpvorderman — CHANGES_REQUESTED** [review]

File: src/main/scala/nl/biopet/tools/gtftorefflat/GtftoRefflat.scala (hunk 1 above)
Comment (ffinfo) [reply src/main/scala/nl/biopet/tools/gtftorefflat/GtftoRefflat.scala]: yes, did some extended testing when creating this tool on this. Some formats are 0 based, some are 1 based. Even worse, sometimes start is 1 based and end is 0 based

File: src/main/scala/nl/biopet/tools/gtftorefflat/GtftoRefflat.scala (hunk 2 above)
Comment (ffinfo) [reply src/main/scala/nl/biopet/tools/gtftorefflat/GtftoRefflat.scala]: same as above

**Review by rhpvorderman — APPROVED** [review]

**Pull request merged**